### ISSUE TITLE
Not possible to edit a model

### ISSUE THREAD

**Issue by abnerrodrigues98** [metabase/metabase#33619]
**Describe the bug**
I can't edit a model

**Logs**
```
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:30-03:00 DEBUG metabase.server.middleware.log POST /api/dashboard/82/dashcard/2944/card/2359/query 202 [ASYNC: completed] 4.8 s (37 chamadas ao banco de dados) Conexões com o banco de dados: 3 / 15 Threads do Jetty: 4 / 50 (21 ocioso, 0 na fila) (218 total de threads ativas) Consultas ativas: 12 (0 na fila); snowflake DB 6 connections: 0/15 (0 threads blocked)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:31-03:00 DEBUG metabase.server.middleware.log POST /api/dashboard/82/dashcard/2755/card/1683/query 202 [ASYNC: completed] 5.1 s (26 chamadas ao banco de dados) Conexões com o banco de dados: 3 / 15 Threads do Jetty: 4 / 50 (21 ocioso, 0 na fila) (217 total de threads ativas) Consultas ativas: 11 (0 na fila); snowflake DB 6 connections: 12/15 (0 threads blocked)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:31-03:00 DEBUG metabase.server.middleware.log POST /api/dashboard/82/dashcard/2759/card/2324/query 202 [ASYNC: completed] 5.4 s (37 chamadas ao banco de dados) Conexões com o banco de dados: 3 / 15 Threads do Jetty: 4 / 50 (21 ocioso, 0 na fila) (215 total de threads ativas) Consultas ativas: 10 (0 na fila); snowflake DB 6 connections: 4/15 (0 threads blocked)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:32-03:00 DEBUG metabase.server.middleware.log POST /api/dashboard/82/dashcard/3373/card/2599/query 202 [ASYNC: completed] 6.3 s (87 chamadas ao banco de dados) Conexões com o banco de dados: 3 / 15 Threads do Jetty: 4 / 50 (21 ocioso, 0 na fila) (210 total de threads ativas) Consultas ativas: 9 (0 na fila); snowflake DB 6 connections: 10/15 (0 threads blocked)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:32-03:00 DEBUG metabase.server.middleware.log POST /api/dashboard/82/dashcard/3901/card/2323/query 202 [ASYNC: completed] 6.5 s (51 chamadas ao banco de dados) Conexões com o banco de dados: 3 / 15 Threads do Jetty: 4 / 50 (21 ocioso, 0 na fila) (209 total de threads ativas) Consultas ativas: 8 (0 na fila); snowflake DB 6 connections: 15/15 (4 threads blocked)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:32-03:00 DEBUG metabase.server.middleware.log POST /api/dashboard/82/dashcard/2749/card/2317/query 202 [ASYNC: completed] 6.9 s (87 chamadas ao banco de dados) Conexões com o banco de dados: 5 / 15 Threads do Jetty: 4 / 50 (21 ocioso, 0 na fila) (208 total de threads ativas) Consultas ativas: 6 (0 na fila); snowflake DB 6 connections: 9/15 (0 threads blocked)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:32-03:00 DEBUG metabase.server.middleware.log POST /api/dashboard/82/dashcard/2756/card/2181/query 202 [ASYNC: completed] 6.8 s (28 chamadas ao banco de dados) Conexões com o banco de dados: 5 / 15 Threads do Jetty: 4 / 50 (21 ocioso, 0 na fila) (208 total de threads ativas) Consultas ativas: 7 (0 na fila); snowflake DB 6 connections: 15/15 (3 threads blocked)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:32-03:00 DEBUG metabase.server.middleware.log POST /api/dashboard/82/dashcard/2752/card/2318/query 202 [ASYNC: completed] 6.9 s (87 chamadas ao banco de dados) Conexões com o banco de dados: 5 / 15 Threads do Jetty: 4 / 50 (21 ocioso, 0 na fila) (207 total de threads ativas) Consultas ativas: 5 (0 na fila); snowflake DB 6 connections: 8/15 (0 threads blocked)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:33-03:00 DEBUG metabase.server.middleware.log POST /api/dashboard/82/dashcard/3133/card/2367/query 202 [ASYNC: completed] 6.9 s (78 chamadas ao banco de dados) Conexões com o banco de dados: 3 / 15 Threads do Jetty: 4 / 50 (21 ocioso, 0 na fila) (205 total de threads ativas) Consultas ativas: 4 (0 na fila); snowflake DB 6 connections: 13/15 (0 threads blocked)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:33-03:00 DEBUG metabase.server.middleware.log POST /api/dashboard/82/dashcard/2753/card/2319/query 202 [ASYNC: completed] 7.6 s (90 chamadas ao banco de dados) Conexões com o banco de dados: 2 / 15 Threads do Jetty: 4 / 50 (21 ocioso, 0 na fila) (205 total de threads ativas) Consultas ativas: 3 (0 na fila); snowflake DB 6 connections: 15/15 (2 threads blocked)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:34-03:00 DEBUG metabase.server.middleware.log POST /api/dashboard/82/dashcard/2945/card/2360/query 202 [ASYNC: completed] 8.1 s (27 chamadas ao banco de dados) Conexões com o banco de dados: 3 / 15 Threads do Jetty: 4 / 50 (21 ocioso, 0 na fila) (205 total de threads ativas) Consultas ativas: 2 (0 na fila); snowflake DB 6 connections: 15/15 (1 threads blocked)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:35-03:00 DEBUG metabase.server.middleware.log GET /api/activity/popular_items 200 13.8 s (10 chamadas ao banco de dados) Conexões com o banco de dados: 0 / 15 Threads do Jetty: 4 / 50 (21 ocioso, 0 na fila) (204 total de threads ativas) Consultas ativas: 2 (0 na fila)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:35-03:00 INFO metabase.server.middleware.exceptions Requisição cancelada antes de finalizar
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:35-03:00 DEBUG metabase.server.middleware.log GET /api/activity/popular_items null 13.8 s (10 chamadas ao banco de dados) Conexões com o banco de dados: 0 / 15 Threads do Jetty: 5 / 50 (20 ocioso, 0 na fila) (204 total de threads ativas) Consultas ativas: 2 (0 na fila)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:36-03:00 DEBUG metabase.server.middleware.log POST /api/dashboard/82/dashcard/2880/card/2334/query 202 [ASYNC: completed] 10.5 s (28 chamadas ao banco de dados) Conexões com o banco de dados: 2 / 15 Threads do Jetty: 3 / 50 (22 ocioso, 0 na fila) (205 total de threads ativas) Consultas ativas: 1 (0 na fila); snowflake DB 6 connections: 15/15 (0 threads blocked)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:37-03:00 DEBUG metabase.server.middleware.log POST /api/dashboard/82/dashcard/2881/card/2313/query 202 [ASYNC: completed] 10.9 s (27 chamadas ao banco de dados) Conexões com o banco de dados: 2 / 15 Threads do Jetty: 3 / 50 (22 ocioso, 0 na fila) (205 total de threads ativas) Consultas ativas: 0 (0 na fila); snowflake DB 6 connections: 14/15 (0 threads blocked)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:48-03:00 DEBUG metabase.server.middleware.log GET /api/session/properties 200 533.7 ms (4 chamadas ao banco de dados) Conexões com o banco de dados: 15 / 15 Threads do Jetty: 25 / 50 (1 ocioso, 0 na fila) (204 total de threads ativas) Consultas ativas: 0 (0 na fila)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:48-03:00 DEBUG metabase.server.middleware.log GET /api/user/current 200 701.6 ms (11 chamadas ao banco de dados) Conexões com o banco de dados: 12 / 15 Threads do Jetty: 24 / 50 (1 ocioso, 0 na fila) (204 total de threads ativas) Consultas ativas: 1 (0 na fila)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:49-03:00 DEBUG metabase.server.middleware.log GET /api/bookmark 200 10.9 ms (1 chamadas ao banco de dados) Conexões com o banco de dados: 15 / 15 Threads do Jetty: 26 / 50 (2 ocioso, 0 na fila) (204 total de threads ativas) Consultas ativas: 6 (0 na fila)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:49-03:00 DEBUG metabase.server.middleware.log GET /api/database 200 81.2 ms (3 chamadas ao banco de dados) Conexões com o banco de dados: 15 / 15 Threads do Jetty: 24 / 50 (2 ocioso, 0 na fila) (204 total de threads ativas) Consultas ativas: 7 (0 na fila)
[a643eb0c-3999-4aa2-b2df-114542fe9d94] 2023-08-30T12:19:49-03:00 WARN metabase.server.middleware.log GET /api/collection/root 403 114.1 ms (2 chamadas ao banco de dados) 
"Desculpe, você não tem permissão para fazer isso."
```

**To Reproduce**
1. Find your model
2. Edit something
3. Try to save 
4. Not possible to save

**Expected behavior**
Should save my edits

**Severity**
Critical. I can't edit anything.

**Additional context**
In logs, I can see the error ""Desculpe, você não tem permissão para fazer isso." or "You don't have permission", but it doens't have any place to change my permissions related to models. Also, I'm administrator

**Metabase Diagnostic Info**

```json
{
  "browser-info": {
    "language": "en-US",
    "platform": "MacIntel",
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.20+8",
    "java.vendor": "Eclipse Adoptium",
    "java.vendor.url": "https://adoptium.net/",
    "java.version": "11.0.20",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.20+8",
    "os.name": "Linux",
    "os.version": "4.14.296-222.539.amzn2.x86_64",
    "user.language": "en",
    "user.timezone": "GMT"
  },
  "metabase-info": {
    "databases": [
      "snowflake",
      "postgres"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "application-database-details": {
      "database": {
        "name": "PostgreSQL",
        "version": "12.14"
      },
      "jdbc-driver": {
        "name": "PostgreSQL JDBC Driver",
        "version": "42.5.4"
      }
    },
    "run-mode": "prod",
    "version": {
      "date": "2023-08-16",
      "tag": "v0.47.0",
      "branch": "release-x.47.x",
      "hash": "682ec2d"
    },
    "settings": {
      "report-timezone": null
    }
  }
}
```

**Comment by paoliniluis** [comment]
Have you already purged your CDN and deleted the browser cache?

**Comment by ignacio-mb** [comment]
Hi @abnerrodrigues98, there is not enough information to reproduce. Kindly try if this is solved in version (47.7).

I will close, but if this still happens please comment.
